src/index.js

```diff
@@ -1 +1,58 @@
-import './index.css';
\ No newline at end of file
+import './index.css';
+
+const form = document.getElementById('form');
+const list = document.getElementById('ListOfScores');
+
+let allScores = '';
+
+// const addGame = async ()=> {
+//      const data = {name: 'fruitninja'}
+
+//     await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/', {
+//         method: 'POST',
+//         headers: {
+//             'Content-Type':'application/json'
+//         },
+//         body: JSON.stringify(data)
+//     }).then(response => response.json())
+//       .then(result =>{
+//         id=result.result.split(' ')[3]
+//         console.log(id)
+//       })
+//       .catch(err=>console.log(err))
+// }
+// addGame()
+
```

Comment:
- [ ] Kindly don't leave the commented piece of code in your project because it makes your codebase looks massy and really unprofessional 😨 Kindly remove this code to make your codebase clean and follow the JavaScript best practices 👍